docs/run_test/understand_results.md

```diff
@@ -0,0 +1,46 @@
+# Understand test results
+
+## Understand log messages
```

Comment:
Let's put this content locally, and find another place to add them.

Reply:
Could you please give more clarification on this?

Reply:
I mean, like what we discussed, the "understand test results" should focus on runner's experience, not how to write the messages. if this part is worth to keep somewhere, keep it locally, and find another place in later refactoring.

Reply:
What do you mean by "locally"? Do you mean I should keep this doc on my computer and save it for fitting in future doc?

Reply:
yes